### PR TITLE
GlobalMarineNet.XGate version 4.3.3

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107508)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation failed with:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
> 'manifests\g\GlobalMarineNet\XGate\1.0.5.27093', expected 
> 'manifests\g\GlobalMarineNet\XGate\4.3.3'

**Comment by stephengillie** [comment]
> Manual Validation failed with:
> 
> > Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual
> > 'manifests\g\GlobalMarineNet\XGate\1.0.5.27093', expected
> > 'manifests\g\GlobalMarineNet\XGate\4.3.3'

I fixed this without checking. Which one of these is the correct version?

**Comment by upintheairsheep** [comment]
> > Manual Validation failed with:
> > > Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual
> > > 'manifests\g\GlobalMarineNet\XGate\1.0.5.27093', expected
> > > 'manifests\g\GlobalMarineNet\XGate\4.3.3'
> 
> I fixed this without checking. Which one of these is the correct version?

The later one

File: manifests/g/GlobalMarineNet/XGate/4.3.3/GlobalMarineNet.XGate.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GlobalMarineNet.XGate
+PackageVersion: 1.0.5.27093
```
Comment (stephengillie) [inline manifests/g/GlobalMarineNet/XGate/4.3.3/GlobalMarineNet.XGate.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.3.3
```

File: manifests/g/GlobalMarineNet/XGate/4.3.3/GlobalMarineNet.XGate.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: GlobalMarineNet.XGate
+PackageVersion: 1.0.5.27093
```
Comment (stephengillie) [inline manifests/g/GlobalMarineNet/XGate/4.3.3/GlobalMarineNet.XGate.yaml:5]:
```suggestion
PackageVersion: 4.3.3
```

**Comment by upintheairsheep** [comment]
???

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ##[error] Installer http://downloads.globalmarinenet.com/downloads/xgsetup4.43.exe 
> ##[error] Result: Failed ##vso[task.logissue type=error] Specified hash doesn't match. 
> Actual hash:   88E81C61CF6F264ED34A8A5F60B1D003894EDC1801AF9C059011F6AB3F3E921B 
> Expected hash: 262ce64d6634c4c7fec57241b26a77ed9bbaff7117f9e01fec2efa3f46b5ce61

File: manifests/g/GlobalMarineNet/XGate/4.3.3/GlobalMarineNet.XGate.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GlobalMarineNet.XGate
+PackageVersion: 4.3.3
+Installers:
+- InstallerUrl: http://downloads.globalmarinenet.com/downloads/xgsetup4.43.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: 262CE64D6634C4C7FEC57241B26A77ED9BBAFF7117F9E01FEC2EFA3F46B5CE61
```
Comment (stephengillie) [inline manifests/g/GlobalMarineNet/XGate/4.3.3/GlobalMarineNet.XGate.installer.yaml:10]:
```suggestion
  InstallerSha256: 88E81C61CF6F264ED34A8A5F60B1D003894EDC1801AF9C059011F6AB3F3E921B
```

**Comment by ItzLevvie** [comment]
[Policy] Interactive-Only-Installer

**Pull request closed without merging**